pom.xml

```diff
@@ -453,11 +453,16 @@
               </includes>
               <trimTrailingWhitespace />
               <endWithNewline />
+              <replaceRegex>
+                <name>Remove wildcard imports</name>
+                <searchRegex>import (static )?\s+\S+\.\*;</searchRegex>
+                <replacement>INVALID IMPORTS (WILDCARD)</replacement>
```

Comment:
Change this to `\n`

Reply:
I think `$` represents end of expression rather than end of line (`\n` or '\r\n`)

Reply:
@brendandburns i tested locally and it seems that the plugin matches the source files by lines so `$` is matching EOL. e.g. adding an `import com.google.guava.*;` line in the middle of the file will be captured.